src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -2852,11 +2852,11 @@ function ExtractPluginProperties([string]$pluginDir, $objectToWriteTo)
         {{
             # Filter the endpoints based on the typeof PowerShell that is
             # executing the cmdlet. {1} in another location indicates that it
-            # is a PowerShell Core endpoint
+            # is a PowerShell endpoint
             if (!($customPluginObject.FileName -match 'system32\\{1}') -AND # WindowsPowerShell
                 !($customPluginObject.FileName -match 'syswow64\\{1}'))     # WOW64 WindowsPowerShell
             {{
-                # Add the PowerShell Core endpoint when running as PowerShell Core
+                # Add the PowerShell endpoint when running as PowerShell
```

Comment:
It seems we it is not full correct too and we need add version.

Reply:
will update